server/proto/management.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
+
+package management;
+
+enum Tee {
+    Sgx = 0;
+    Tdx = 1;
+    SevSnp = 2;
+    Sample = 3;
+}
+
+message GetPolicyRequest {
+    Tee tee = 1;
+    optional string user = 2;
+}
+message GetPolicyResponse {
+    bytes content = 1;
+}
+
+message GetReferenceDataRequest {
+    Tee tee = 1;
+    optional string user = 2;
+}
+message GetReferenceDataResponse {
+    bytes content = 1;
+}
+
+message SetPolicyRequest {
+    Tee tee = 1;
+    optional string user = 2;
+    bytes content = 3;
+}
+message SetPolicyResponse {}
+
+message SetReferenceDataRequest {
+    Tee tee = 1;
+    optional string user = 2;
+    bytes content = 3;
+}
+message SetReferenceDataResponse {}
+
+message RestoreDefaultPolicyRequest {
+    Tee tee = 1;
+    optional string user = 2;
+}
+message RestoreDefaultPolicyResponse {}
+
+message RestoreDefaultReferenceDataRequest {
+    Tee tee = 1;
+    optional string user = 2;
+}
+message RestoreDefaultReferenceDataResponse {}
+
+message TestOpaRequest {
+    bytes policy = 1;
+    bytes reference = 2;
+    bytes input = 3;
+}
+message TestOpaResponse {
+    bytes status = 1;
+}
+
+service ManagementService {
+    rpc GetPolicy(GetPolicyRequest) returns (GetPolicyResponse) {};
+    rpc GetReferenceData(GetReferenceDataRequest) returns (GetReferenceDataResponse) {};
+    rpc SetPolicy(SetPolicyRequest) returns (SetPolicyResponse) {};
+    rpc SetReferenceData(SetReferenceDataRequest) returns (SetReferenceDataResponse) {};
+    rpc RestoreDefaultPolicy(RestoreDefaultPolicyRequest) returns (RestoreDefaultPolicyResponse) {};
```

Comment:
Could we comment on that part of the API? What is a `Default` policy? Is it specific to a user? Where is it defined?

Reply:
The default policy is integrated inside the Attestation Service Crate, please refere the following code of sample TEE:
```
    let policy = policy.unwrap_or_else(|| std::include_str!(default_policy!()).to_string());
    let reference_data = reference_data
        .unwrap_or_else(|| std::include_str!(default_reference_data!()).to_string());
```
For sample TEE, it's default Policy is defined in: `lib/src/core/verifier/sample/policy.rego`

It is specific to different type of TEEs, and each user's default Policy/Reference Data are the same.